### ISSUE TITLE
BATCH IMPORT: Create Batch of works button fails

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/epigaea#641]
**STEPS TO REPRODUCE**
1. Login as an administrator
1. Go to the admin dashboard
1. Click on the "Works" link in the left
1. Click on "Create batch of works" button
1. Select "Audio" and click on "Create Work"

**EXPECTED BEHAVIOR**
I should probably be take to the new object import page when I click "Create batch of works"

**ACTUAL BEHAVIOR**
I get an error

**Comment by little9** [comment]
@mark-dce @no-reply @bess  this is because Hyrax has form class that `BatchUploadForm` inherits from that has its own set of metadata terms that is different from ours: 

https://github.com/samvera/hyrax/blob/8e785de57f474629ce8757859628800626984bea/app/forms/hyrax/forms/work_form.rb

I'm not sure the best way to override this stuff

**Comment by little9** [comment]
Nevermind, I got it.

**Comment by val99erie** [comment]
Re-opening issue after some discussion on slack.  Expect a new PR soon

**Comment by no-reply** [comment]
@mark-dce can you confirm that this is closed by #684?

**Comment by mark-dce** [comment]
Yes, this looks sufficient.  As a bonus, it pretty much replaces what Tufts used to have as "Template Import" functionality - load a bunch of files as individual objects and give them the same descriptive metadata.  Thanks @little9 and @no-reply
